src/EnergyPlus/SizingManager.cc

```diff
@@ -3166,8 +3166,20 @@ namespace SizingManager {
                                 ErrorsFound = true;
                             }
                         }
-                        ZoneSizingInput(ZoneSizIndex).DOASLowSetpoint = rNumericArgs(17);
-                        ZoneSizingInput(ZoneSizIndex).DOASHighSetpoint = rNumericArgs(18);
+                        // blank fields are set to autosize
+                        if ( lNumericFieldBlanks( 17 ) ) {
+                            ZoneSizingInput(ZoneSizIndex).DOASLowSetpoint = DataSizing::AutoSize;
+                        } else
+                        {
+                            ZoneSizingInput(ZoneSizIndex).DOASLowSetpoint = rNumericArgs(17);
+                        }
+                        if ( lNumericFieldBlanks( 18 ) ) {
+                            ZoneSizingInput(ZoneSizIndex).DOASHighSetpoint = DataSizing::AutoSize;
+                        } else
+                        {
+                            ZoneSizingInput(ZoneSizIndex).DOASHighSetpoint = rNumericArgs(18);
+                        }
+
```

Comment:
If these two fields are left blank and ZoneSizingInput(*).AccountForDOAS is "true" ("Account for Dedicated Outdoor Air System" is set to "YES"), then the "DOASLowSetpoint" and "DOASHighSetpoint" values are set to Autosize.